src/AutoMapper/QueryableExtensions/Impl/QueryMapperHelper.cs

```diff
@@ -24,7 +24,7 @@ public static PropertyMap GetPropertyMap(this IConfigurationProvider config, Mem
 
         public static TypeMap CheckIfMapExists(this IConfigurationProvider config, Type sourceType, Type destinationType)
         {
-            var typeMap = config.FindTypeMapFor(sourceType, destinationType);
+            var typeMap = config.ResolveTypeMap(sourceType, destinationType);
```

Comment:
But it seems to be used only for QueryableExtensions, so I was probably thinking of smth else :) Clearly you usually want Resolve, not Find.